data/json/furniture_and_terrain/furniture-medical.json

```diff
@@ -492,7 +492,7 @@
     "type": "furniture",
     "id": "f_MS",
     "name": "mass spectrometer",
-    "description": "Inside this thing is a carefully balanced set of electric field generators that can precisely separate ionized particles based on their charge-to-mass ratio, firing them into a detector that measures the exact mass of the particle hitting it.  On the outside, it looks like a very boring white box.",
+    "description": "Inside this large white box is a carefully balanced set of electric field generators that can precisely separate ionized particles based on their charge-to-mass ratio, firing them into a detector that measures the exact mass of the particle hitting it.  Invaluable for particle research and other advanced sciences, it's not as useful anymore.",
```

Comment:
I am no expert for MS but in "normal" chemistry a MS is mostly used for analytical chemistry and not for particle research.
So I would change it to 
"Invaluable for analytical chemistry and other advanced sciences, it's not as useful anymore.","